libs/dataset_deployer.py

```diff
@@ -67,9 +70,54 @@ def upload_csv(key_name: str, csv: str, output_dir: str):
     _logger.info(f"Generated csv for {key_name}")
 
 
-def upload_json(key_name, json: str, output_dir: str): 
+def flatten_dict(data: dict, level_separator: str = '.') -> dict:
```

Comment:
is there no way this dictionary can have lists in it already?

Reply:
So I think it can have lists... as of now the list will just be one column. Can raise an error if we think that's a problem.

Reply:
cool i think that's is okay (I mean i think some slightly complicated things can happen like a list of dictionaries), but maybe we deal with that if/when it happens